### PR TITLE
Itm 127 docs test in gradle

### PR DESCRIPTION
**Pull request by kdrumm**
Added to tasks to copy tests, docs and REAMDE-API.md.

### CODE REVIEW

File: swagger_client/__init__.py [hunk 1]
```diff
@@ -21,16 +21,27 @@
 from swagger_client.configuration import Configuration
 # import models into sdk package
 from swagger_client.models.action import Action
+from swagger_client.models.action_type import ActionType
```
Comment (nextcen-dgemoets) [inline swagger_client/__init__.py:24]: How were these changes not already in the `development` tree?  Did we forget something in the previous ticket?

**Review by nextcen-dgemoets — APPROVED** [review]
Shoot-- such an easy fix after all (to remove `components` from the `code` block).

Thanks for fixing!  Changes look good to me.  Did have the question about `swagger_client/__init__.py`, though.

File: swagger_client/__init__.py (hunk 1 above)
Comment (kdrumm) [reply swagger_client/__init__.py:24]: That is an excellent question that I do not know the answer to!  We must have done something wrong as they should be in there.

**Comment by kdrumm** [comment]
> Shoot-- such an easy fix after all (to remove `components` from the `code` block).
> 
> Thanks for fixing! Changes look good to me. Did have the question about `swagger_client/__init__.py`, though.

It was easy, but quite hard to find.  Our use of gradle and the documentation doesn't seem to be in alignment.  Of course, it could be my lack of recent gradle usage and super rusty skills there.

**Review by brianpippin — APPROVED** [review]

**Pull request merged**